### PR TITLE
144 implement a reusable help component

### LINKED ISSUE #144: Implement a reusable Help component
Description:

## Problem
We need a way to consistently display FAQs on each page of our application to help users understand how to use each section.

## Proposed Solution
Implement a reusable 'Help' component in React and TypeScript. This component should be a popup, tooltip, or other UI element that provides helpful information to users. It should be importable and usable on any page of the application.

## Details
- The component should accept an array of FAQs as props. Each FAQ will have a 'question' and an 'answer'.
- The component should display each question and answer in a consistent and readable format.
- Consider accessibility concerns, such as providing alternative text for any visual content in the help popup.

## Acceptance Criteria
- The component displays the correct FAQs passed in as props.
- The component is used on multiple pages and displays correctly in each instance.
- The component meets all accessibility standards.

Please let me know if you need any more information. Thank you!

### PR DESCRIPTION
**Pull request by glenntanjoh**
Resolves #144 

Made style and grammatical error fixes and updated grammar in jest test

### CODE REVIEW

File: frontend/src/components/HelpPopup/HelpPopup.tsx [hunk 1]
```diff
@@ -114,7 +73,12 @@ function HelpPopup(): React.ReactElement | null {
         }
     }
 
-    getRenders();
+    const listOfTitles = ["File Upload", "Processed Files", "Predicted Files"];
+    const listOfDescriptions = [
+        "Step 1: Drag and drop your files into the upload box. Once uploaded, your files will appear in the Uploaded Files section on the right. You can select the file you want to process and click the process button. Once you have processed your data it will be saved. You do not need to process it again. Click Go To Processed Files to move to the processed data menu.",
```
Comment (mpk467) [inline frontend/src/components/HelpPopup/HelpPopup.tsx]: change to en-dash

**Review by mpk467 — COMMENTED** [review]
I ran the text through Grammarly. LGTM!

Just change : to - in step 1

File: frontend/src/components/HelpPopup/HelpPopup.test.tsx [hunk 2]
```diff
@@ -45,16 +45,13 @@ test("TID 3.1. Renders HelpPopup component", () => {
         } else {
             userEvent.click(button);
             expect(getByTestId("Paragraph 0").getAttribute("values")).toEqual("opened");
-            expect(getByTestId("button 0").innerHTML).toEqual("^");
             getByText(
-                "Step 1: Drag and drop your files into the upload box. Once uploaded your files will appear in the Uploaded Files section on the right. You can select the file you want to process and click the process button. Once you have processed your data it will be saved. You do not need to process it again. Click Go To Processed Files to move to the processed data menu.",
+                "Step 1: Drag and drop your files into the upload box. Once uploaded, your files will appear in the Uploaded Files section on the right. You can select the file you want to process and click the process button. Once you have processed your data it will be saved. You do not need to process it again. Click Go To Processed Files to move to the processed data menu.",
```
Comment (mpk467) [inline frontend/src/components/HelpPopup/HelpPopup.test.tsx]: "Step 1 - " to be consistent with step 2 and 3

**Review by ralphgregorio — APPROVED** [review]
looks good

**Pull request merged**